### PR TITLE
add datetime admin 

### PR DESCRIPTION
**Pull request by zhuyaliang**
Added new time and date management services

### CODE REVIEW

**Comment by raveit65** [comment]
@zhuyaliang 
1. Can you please commit *.png and *.po files in 2 extra commits?
It is very hard to take a look and scroll in such a big commit with thousands of lines only for images and po files. Here is a guide how to split a commit with the `git rebase` command.
https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits
And patching a rpm installer file isn't possible with image files.

2. You don't need to commit *.mo files.
Those files will create during building the package.
And in which dir the language files will be installed?

**Comment by raveit65** [comment]
Beside from that it seems to working fine and i don't get any warning :)
Bildschirmfoto zu 2019-06-07 12-12-54

**Comment by raveit65** [comment]
Ntp or ntpdate aren't installed here on fedora-30. So i am still wondering how time and timezone will be synced?

**Comment by rbuj** [comment]
@raveit65 
By using [timedatectl](https://www.freedesktop.org/software/systemd/man/timedatectl.html) utility, which is new in [Fedora 30](https://docs.fedoraproject.org/en-US/fedora/rawhide/system-administrators-guide/basic-system-configuration/Configuring_the_Date_and_Time/) and is part of systemd.

**Comment by raveit65** [comment]
@rbuj 
Than i am still wondering why this service is masked in my f30 installation.
```
systemd-time-wait-sync.service                   disabled       
systemd-timedated.service                        masked         
systemd-timesyncd.service                        disabled
```
Or was this masked when f30 was rawhide?
I started very early with using rawhide/f30 if plan to use a fedora release.

**Comment by rbuj** [comment]
`timedatectl` enables/disables `chronyd.service`
```shell
[robert@localhost ~]$ sudo timedatectl set-ntp no && systemctl list-unit-files > 1
[robert@localhost ~]$ sudo timedatectl set-ntp yes && systemctl list-unit-files > 2
[robert@localhost ~]$ diff 1 2
57c57
< chronyd.service                                      disabled       
---
> chronyd.service                                      enabled        
```

**Comment by raveit65** [comment]
ok, i 've enabled systemd-timesyncd.service unit.
Now:
```
[root@mother rave]# timedatectl timesync-status
       Server: 62.116.130.3 (0.fedora.pool.ntp.org)
Poll interval: 1min 4s (min: 32s; max 34min 8s)
         Leap: normal
      Version: 4
      Stratum: 2
    Reference: C035676C
    Precision: 1us (-23)
Root distance: 31.463ms (max: 5s)
       Offset: -2.930ms
        Delay: 29.999ms
       Jitter: 0
 Packet count: 1
    Frequency: +48,588ppm


[root@mother rave]# timedatectl
               Local time: Fr 2019-06-07 14:06:49 CEST
           Universal time: Fr 2019-06-07 12:06:49 UTC
                 RTC time: Fr 2019-06-07 14:06:50
                Time zone: Europe/Berlin (CEST, +0200)
System clock synchronized: yes
              NTP service: inactive
          RTC in local TZ: no

```

**Comment by rbuj** [comment]
@raveit65 
You don't need to enable timesync daemon, since you have chrony installed on F30+. systemd-timesyncd (SNTP client) and chrony (NTP client), both are daemons for synchronizing the system clock across the network.
```shell
$ LANG=C systemctl status chronyd
```

File: capplets/time-admin/data/mate-time-admin.desktop.in [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Name=Time And DateManager
```
Comment (rbuj) [inline capplets/time-admin/data/mate-time-admin.desktop.in]:
```patch
diff --git a/capplets/time-admin/data/mate-time-admin.desktop.in b/capplets/time-admin/data/mate-time-admin.desktop.in
index 26bff82..3829dc9 100644
--- a/capplets/time-admin/data/mate-time-admin.desktop.in
+++ b/capplets/time-admin/data/mate-time-admin.desktop.in
@@ -1,13 +1,11 @@
 [Desktop Entry]
-Name=Time And DateManager
-Name[zh_CN]=时间和日期管理
-Comment=Manage Time And Date 
-Comment[zh_CN]=管理时间和日期。
+_Name=Time And Date
+_Comment=Manage time and date
 Terminal=false
 Type=Application
 Exec=mate-time-admin
 Icon=mate-times-admin
 StartupNotify=true
-Categories=GTK;Settings;X-MATE-PersonalSettings;
-Keywords=mate-control-center;MATE;personal;information;
+Categories=GTK;Settings;HardwareSettings;
+Keywords=mate-control-center;MATE;clock;date;time;preferences;
 OnlyShowIn=MATE;
```

Comment (rbuj) [reply capplets/time-admin/data/mate-time-admin.desktop.in]: Don't use Title Case for Comment in mate-time-admin.desktop.in:
```shell
$ git grep "Comment="
capplets/about-me/mate-about-me.desktop.in:_Comment=Set your personal information
capplets/accessibility/at-properties/mate-at-properties.desktop.in:_Comment=Choose which accessibility features to enable when you log in
capplets/appearance/data/mate-appearance-properties.desktop.in:_Comment=Customize the look of the desktop
capplets/appearance/data/mate-theme-installer.desktop.in:_Comment=Installs themes packages for various parts of the desktop
capplets/appearance/theme-save.c:               "Comment=%s\n"
capplets/default-applications/mate-default-applications-properties.desktop.in:_Comment=Select your default applications
capplets/display/mate-display-properties.desktop.in:_Comment=Change resolution and position of monitors and projectors
capplets/keybindings/mate-keybinding.desktop.in:_Comment=Assign shortcut keys to commands
capplets/keyboard/mate-keyboard.desktop.in:_Comment=Set your keyboard preferences
capplets/mouse/mate-settings-mouse.desktop.in:_Comment=Set your mouse preferences
capplets/network/mate-network-properties.desktop.in:_Comment=Set your network proxy preferences
capplets/time-admin/data/mate-time-admin.desktop.in:Comment=Manage Time And Date 
capplets/windows/mate-window-properties.desktop.in:_Comment=Set your window properties
font-viewer/mate-font-viewer.desktop.in:_Comment=View fonts on your system
shell/matecc.desktop.in:_Comment=Configure MATE settings
```
Name and Comment should start with underscore for l10n. Test:
```shell
$ ./autogen.sh --prefix=/usr
$ ./makepot
$ grep "Time And Date Manager" mate-control-center.pot
(changes)
$ ./makepot
$ grep "Time And Date Manager" mate-control-center.pot
```

Remove trailing space in Comment.


*.desktop.in files shouldn't include Name[LOCALE]=TRANSLATION, Comment[LOCALE]=TRANSLATION

**Pull request closed without merging**